attribution-graphs-frontend/attribution_graph/init-cg-link-graph.js

```
@@ -0,0 +1,339 @@
+    })
+  })
+  nodes.forEach(d => {
+    const streamIdx = d.layer == "E" ? 0 : 1 + d.streamIdx;
+    // const streamIdx = d.streamIdx;
+    const xOffset = d.xOffset;
+    d.pos = [
+      c.x(d.ctx_idx) + xOffset,
+      c.y(streamIdx) + c.y.bandwidth()/2 + d.yOffset
+    ]
+  })
+  const filt = nodes.filter(d => d.pos.some(x => !isFinite(x)))
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)